firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/TesterApiHttpClient.java

```diff
@@ -48,6 +52,11 @@ class TesterApiHttpClient {
   private static final String X_ANDROID_CERT_HEADER_KEY = "X-Android-Cert";
   // Format of "X-Client-Version": "{ClientId}/{ClientVersion}"
   private static final String X_CLIENT_VERSION_HEADER_KEY = "X-Client-Version";
+  private static final String X_GOOG_UPLOAD_PROTOCOL_HEADER = "X-Goog-Upload-Protocol";
+  private static final String X_GOOG_UPLOAD_PROTOCOL_RAW = "raw";
+  private static final String X_GOOG_UPLOAD_FILE_NAME_HEADER = "X-Goog-Upload-File-Name";
+  private static final String X_GOOG_UPLOAD_FILE_NAME = "screenshot.png";
+  private static final String UTF_8 = "UTF-8";
```

Comment:
why not use `StandardCharsets.UTF_8`?

Reply:
It's not available unfortunately: `Field requires API level 19 (current min is 16): java.nio.charset.StandardCharsets#UTF_8`. I added a comment.

Apparently I can use it in a test though.